src/status_im2/contexts/quo_preview/avatars/group_avatar.cljs

```diff
@@ -39,7 +42,12 @@
          {:padding-vertical 60
```

Comment:
https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#always-add-styles-inside-the-style-key